init_test_data.sql

```diff
@@ -2,31 +2,38 @@ USE data;
 
 -- Create initial tables
 
+CREATE TABLE geography (
+    geography_id serial primary key,
+    abreviation char(2),
+    full_name varchar not null
+);
+
 CREATE TABLE batch (
-    batch_id INT NOT NULL AUTO_INCREMENT,
-    push_time DATETIME,
+    batch_id serial primary key,
+    push_time timestamptz not null,
+    publish_time timestamptz,
```

Comment:
what's the difference between push_time and publish_time?

Reply:
based on Josh's work, I was thinking it could make sense to have push_time be the time the batch was pushed/created and publish_time be the time the batch actually got published. Maybe we can call them `created_at` and `published_at` for clarity?